suite-common/wallet-core/src/send/sendFormRippleStellarThunks.ts

```diff
@@ -8,15 +8,23 @@ import {
     PrecomposedTransaction,
 } from '@suite-common/wallet-types';
 import {
+    asAmountUnit,
     calculateMax,
     calculateTotal,
     formatNetworkAmount,
     getExternalComposeOutput,
     isTestnet,
     networkAmountToSmallestUnit,
+    unitsToSubunits,
 } from '@suite-common/wallet-utils';
 import { buildSendTransaction, toStroops } from '@trezor/blockchain-link-utils/src/stellar';
-import TrezorConnect, { FeeLevel, RipplePayment, StellarOperation } from '@trezor/connect';
+import TrezorConnect, {
+    FeeLevel,
+    RipplePayment,
+    StellarOperation,
+    TokenInfo,
+} from '@trezor/connect';
+import { StellarAssetType } from '@trezor/protobuf/src/messages';
```

Comment:
The same (here and other places), it would be nice to export this and 
```suggestion
import { StellarAssetType } from '@trezor/protobuf';
```